src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -138,47 +138,33 @@ class BaseExpensiTextInput extends Component {
     }
 
     render() {
-        const {
-            label,
-            value,
-            placeholder,
-            errorText,
-            hasError,
-            containerStyles,
-            inputStyle,
-            ignoreLabelTranslateX,
-            innerRef,
-            autoFocus,
-            multiline,
-            ...inputProps
-        } = this.props;
-
-        const hasLabel = Boolean(label.length);
+        const inputProps = _.omit(this.props, _.keys(propTypes));
```

Comment:
Seeing this again and maybe it's better to pick out which ones we _do_ want to include in `inputProps`?